daprdocs/content/en/operations/hosting/kubernetes/kubernetes-persisting-scheduler.md

```diff
@@ -6,11 +6,11 @@ weight: 50000
 description: "Configure Scheduler to persist its database to make it resilient to restarts"
 ---
 
-The [Scheduler]({{< ref scheduler.md >}}) service is responsible for writing jobs to its embedded database and scheduling them for execution.
-By default, the Scheduler service database writes this data to an in-memory ephemeral tempfs volume, meaning that **this data is not persisted across restarts**. Job data will be lost during these events.
-
-To make the Scheduler data resilient to restarts, a persistent volume must be mounted to the Scheduler `StatefulSet`.
+The [Scheduler]({{< ref scheduler.md >}}) service is responsible for writing jobs to its embedded database and scheduling them for execution. On installation, a Persistent Volume Claim (PVC) is automatically created on the default storage class, providing durable storage for the Scheduler database and making it resilient to restarts or cluster failure. 
```

Comment:
Please, mention somewhere that the default size is 1Gi and it can be changed. It is recommended to use 10Gi for production.

Reply:
Currently, the docs suggest 64GB, should we update that, or leave it as is?